programs/cleos/main.cpp

```diff
@@ -224,6 +230,9 @@ void add_standard_transaction_options(CLI::App* cmd, string default_permission =
    cmd->add_option("--max-net-usage", tx_max_net_usage, localized("set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)"));
 
    cmd->add_option("--delay-sec", delaysec, localized("set the delay_sec seconds, defaults to 0s"));
+   cmd->add_option("-t,--return-failure-trace", tx_rtn_failure_trace, localized("Return partial traces on failed transactions"));
+   cmd->add_option("--retry", tx_retry, localized("Request node to retry transaction until irreversible or in a block of height --retry-num-blocks"));
```

Comment:
What do you think about making this one --retry-irreversible and then making them exclusive? It will make the comments clearer. Otherwise the one below is confusing since you could never set --retry-num-blocks and ever end up getting an irreversible retry.

Reply:
ok. Done.